package/MDAnalysis/coordinates/ParmEd.py

```diff
@@ -211,9 +212,21 @@ def convert(self, obj):
         atom_kwargs = []
         for atom, name, resname, xyz, vel in zip(ag_or_ts, names, resnames, 
                                                  positions, velocities):
-            akwargs = {'name': name}
-            chain_seg = {'segid': atom.segid}
-            for attrname in ('mass', 'charge', 'type',
+            try:
```

Comment:
Are these changes necessary if we're killing `ts`?

Reply:
I guess I'm anticipating changes in API that don't guarantee resids/masses/whatever, but we can cross that bridge when we come to it. I'll revert it.